cac-prod.yml

```diff
@@ -178,6 +178,7 @@ unclassified:
       - name: 'Infrastructure'
         includeInChangesets: false
         defaultVersion: master
+        allowVersionOverride: false
```

Comment:
the new name may want a change too on line 189

Reply:
maybe, unless we want to keep it for testing?

Reply:
On second thoughts, we'll disallow version override throughout - then if we need to reenable for testing we can just do it on the unused Pipeline one

Reply:
Makes sense